.CI/build_all

```diff
@@ -10,6 +10,7 @@ if [ -z "$GH_TOKEN" ]; then
     python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
+    conda config --add channels r
```

Comment:
Sorry @msarahan your ~~name~~ GitHub handle is always the first one to come to mind. I will try to forget you a little bit :wink: 

@mingwandroid I am looking forward to hear your thoughts on this topic.